adoc/admin-updates.adoc

```diff
@@ -19,15 +19,19 @@ Updating of {kube} components is handled via `skuba`.
 
 The general procedure should look like this:
 
-. Update the packages and reboot your management workstation to get all the latest changes to skuba and it's dependencies
-. Run `skuba addon upgrade apply` on the management workstation
-.
-. Check if there are addon upgrades available for the current cluster
-. Upgrade all master nodes
-. Upgrade all worker nodes
-. Check if new addons are available for the new version
-.. If necessary upgrade master nodes again
-.. If necessary upgrade worker nodes again
+. Update the packages and reboot your management workstation to get all the latest changes to skuba, helm and their dependencies
+. Run `skuba addon upgrade plan` and then `skuba add upgrade apply` on the management workstation
+. Apply all the configuration files that you modified for addons, the upgrade will have reset the configurations to defaults
```

Comment:
We really have to explain how to do this! I.e. re-applying the existing and modified dex.yaml will i.e. just "reset" the deployment to 4.0.1 settings and also will have a wrong secret that does not match to the gangway deployment anymore.
Current manual process -> run skuba init with original parameters for the cluster in a temp directory, merge the new dex.yaml with the old dex.yaml and the kubectl replace -f ... -force

Reply:
@ereslibre Can you formulate a process that can be applied to any deployment and not just specifically to dex? Are we planning to do take care of this config updating automatically at some point? This looks like a huge source for errors to me.

Reply:
https://github.com/SUSE/caasp-rfc/pull/46 is tackling this issue. I have a PR almost ready, but I have to go through some things yet. The PR is not yet open: https://github.com/SUSE/skuba/compare/master...ereslibre:kustomize.

I'll check if we can propose a solution in the meantime.

Reply:
This will now be handled in a separate issue #692